internal_testsuites/golang/testsuite/startosis_run_sh_task_test/run_task_sh_task_test.go

```diff
@@ -63,7 +63,7 @@ func TestStarlark_RunshTaskFileArtifact(t *testing.T) {
 func TestStarlark_RunshTaskFileArtifactFailure(t *testing.T) {
 	ctx := context.Background()
 	runResult, _ := test_helpers.SetupSimpleEnclaveAndRunScript(t, ctx, runshTest, runshStarlarkFileArtifactFailure)
-	expectedErrorMessage := "error occurred and shell command: \"cat /tmp/kurtosis.txt\" exited with code 1 with output \"cat: can't open '/tmp/kurtosis.txt': No such file or directory\\n"
+	expectedErrorMessage := "cat: can't open '/tmp/kurtosis.txt': No such file or directory"
```

Comment:
From @Peeeekay : "Because the error also prints out the path of the file; which is non deterministic as it can change depending upon whether the tests are runnin locally or on ci"